catalyst-gateway/bin/src/settings/cassandra_db.rs

```diff
@@ -91,6 +103,15 @@ impl EnvVars {
                 MIN_BATCH_SIZE,
                 MAX_BATCH_SIZE,
             ),
+            region_1: StringEnvVar::new(
+                &format!("CASSANDRA_{name}_REGION_1"),
+                "eu-central-1".into(),
+            ),
+            region_2: StringEnvVar::new(&format!("CASSANDRA_{name}_REGION_2"), "eu-west-1".into()),
+            replication_factor: StringEnvVar::new(
+                &format!("CASSANDRA_{name}_REPLICATION_FACTOR"),
```

Comment:
Replication factor should default to 1.
If something else is required in production, it should be set in production.
Otherwise the default behaviour in development is to store three times the amount of data required for testing.
We should not encode production choices into the default configs.

Reply:
As discussed, a replication of 3 is required for aws. I have removed production choices and replaced with generic options.